bindings/java/src/main/java/org/ucx/jucx/NativeLibs.java

```diff
@@ -99,13 +100,20 @@ private static void extractUCTLibs() {
         }
 
         uctLibs.forEach(filePath -> {
+            if (!filePath.getFileName().toString().startsWith("lib")){
```

Comment:
i think better to check ```.endsWith(".so")``` to detect shared lib.
folder can contain libsmth.la or similar meta data.